zerver/tests/webhooks/test_bitbucket.py

```diff
@@ -119,23 +119,23 @@ def test_bitbucket2_on_pull_request_merged_event(self):
 
     def test_bitbucket2_on_pull_request_comment_created_event(self):
         # type: () -> None
-        expected_message = u"User Tomasz(login: kolaszek) created [comment](https://bitbucket.org/kolaszek/repository-name/pull-requests/3/_/diff#comment-20576503 in [\"new commit\" pull request](https://bitbucket.org/kolaszek/repository-name/pull-requests/3)"
+        expected_message = u"kolaszek [commented](https://bitbucket.org/kolaszek/repository-name/pull-requests/3/_/diff#comment-20576503) [PR](https://bitbucket.org/kolaszek/repository-name/pull-requests/3)\n\n~~~ quote\nComment1\n~~~"
```

Comment:
sorry, without on its like polish-english ;-)

As to PR number - we decided to keep it only in a subject. Are you sure I should add it (then I'll add it to all integrations, prolly as a separate commit)